backend/src/main/java/service/User.java

```diff
@@ -1,3 +1,5 @@
+package service;
```

Comment:
Model classes should not be in service packs. It's better to create a separate "model" package or keep all classes responsible for the user in the "user" package

Reply:
Created "model" package. User moved to it.